.github/workflows/ui-tests-common.yml

```
@@ -0,0 +1,103 @@
+name: UI Tests Common
+
+on:
+  workflow_call:
+    inputs:
+      fastlane_command:
+        description: "The fastlane command to run"
+        required: true
+        type: string
+      xcresult_name:
+        description: "Name for the xcresult artifact"
+        required: true
+        type: string
+      xcode_version:
+        description: "Xcode version"
+        required: true
+        type: string
+      build_with_make:
+        description: "Build with make"
+        required: false
+        default: false
+        type: boolean
+      build_with_script:
```

could we make this name more obvious that it triggers an xcframework build?
```suggestion
      build_xcframework:
```